bin/pyenv-pip-update

```diff
@@ -27,7 +27,7 @@ then # If no argument is given, update libraries using pip
             conda update --all
         elif pip >/dev/null 2>&1
         then
-            stale_packages="$(pip list --format=freeze --outdated | awk -F '==' '{printf "%s ", $1}')"
+            stale_packages="$(pip --disable-pip-version-check list --outdated | awk 'NR>3 {print $1}')"
```

Comment:
```suggestion
            stale_packages="$(pip --disable-pip-version-check list --outdated | awk 'NR>2 {print $1}')"
```

I think `NR>2` is correct because you want to skip "two" lines.

Reply:
You are right of course. I am sorry I did such a bad job.